Jint.Tests/Runtime/ErrorTests.cs

```diff
@@ -282,6 +283,21 @@ public void StackTraceCollectedForImmediatelyInvokedFunctionExpression()
             EqualIgnoringNewLineDifferences(expected, ex.ToString());
         }
 
+        [Fact]
+        public void StackTraceIsForOriginalException()
+        {
+            var engine = new Engine();
+            engine.SetValue("HelloWorld", new HelloWorld());
+            const string script = @"HelloWorld.ThrowException();";
+
+            var ex = Assert.Throws<DivideByZeroException>(() => engine.Execute(script));
+
+            const string expected = @"System.DivideByZeroException: Attempted to divide by zero.
```

Comment:
I would just check for the type of exception. Checking on the string is too brittle.

Reply:
Isn't that a stupid comment ;) Just look for some part of the string that you think should be there, but not the exact one, the filenames or lines might change

Reply:
I agree the string is brittle, but if we don't somehow string check the stacktrace then it won't properly test my PR is effective.  Should we just remove it regardless?

Reply:
fixed